tasks/task_manual_content_scan.c

```diff
@@ -0,0 +1,354 @@
+/*  RetroArch - A frontend for libretro.
+ *  Copyright (C) 2011-2017 - Daniel De Matteis
+ *  Copyright (C) 2014-2017 - Jean-André Santoni
+ *  Copyright (C) 2016-2019 - Brad Parker
+ *
```

Comment:
Add your name here too, it's your addition :)

Reply:
Ha! Yes, you're right :)

I've added quite a few source files, but I've always just copy/pasted the header comment without really looking at it. I need to go through all of them and update the copyright text!